Source/Core/VideoCommon/TextureConversionShader.cpp

```diff
@@ -53,7 +53,7 @@ u16 GetEncodedSampleCount(u32 format)
 
 // block dimensions : widthStride, heightStride
 // texture dims : width, height, x offset, y offset
-static void WriteSwizzler(char*& p, u32 format, API_TYPE ApiType)
+static void WriteSwizzler(char*& p, u32 format, API_TYPE ApiType, int numInputSamples)
```

Comment:
A global is fine; the code already does that.  Or you could make TextureConversionShader a class, if you prefer; that's probably a little more clean.